spec/spec_helper.rb

```diff
@@ -1,6 +1,5 @@
-# frozen_string_literal: true
-Dir[File.join(Dir.pwd, 'spec', 'support','**', '*.rb')].each {|f| require f}
-$LOAD_PATH <<  File.join(Dir.pwd, 'spec', 'fixtures')
+Dir[File.join(Dir.pwd, 'spec', 'inquisition', 'support', '**', '*.rb')].each { |f| require f }
```

Comment:
This should happen after `require` of the gem, also this line can be replaced by the next line:
```ruby
Dir.glob(File.expand_path('support/**/*.rb', __dir__), &method(:require))
```